src/routes/landing.js

```diff
@@ -21,6 +21,13 @@ class Landing extends React.Component {
             />
           </FullWidthContainer>
 
+          <FullWidthContainer flex={columnContainerFlex}>
+            <Button
+              title='Skip search and go to results' //Defenetly temporary. Eventually the search page will lead to the results page.
```

Comment:
spelling, "Definitely"
Even though it's a temporary comment, it's good to keep it professional